Fenner_William-DFBT/js/index.js

```diff
@@ -4,10 +4,15 @@
 
 function form_builder(attach, ...form_elements){
 
+    //Create a form element
     var form = document.createElement('form');
 
+    //Loop through all form elements from an array
     for(var i=0; i<form_elements.length; i++){
```

Comment:
Later on in this loop the variables with only single characters become confusing, maybe instead use descriptive names for the variables.